wandb/proto/wandb_internal.proto

```diff
@@ -449,6 +449,7 @@ message ArtifactManifest {
   string storage_policy = 2;
   repeated StoragePolicyConfigItem storage_policy_config = 3;
   repeated ArtifactManifestEntry contents = 4;
+  string manifest_file_path = 5;
```

Comment:
Documenting protos helps a lot with debugging and refactoring later. In this case, it would help to clarify that only one of `contents` or `manifest_file_path` is used and to mention the format of the manifest file.